sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubListenerTests.cs

```diff
@@ -222,7 +222,7 @@ public void GetMonitor_ReturnsExpectedValue()
                                     host,
                                     false,
                                     consumerClientMock.Object,
-                                    Mock.Of<BlobsCheckpointStore>(),
+                                    Mock.Of<global::Azure.Messaging.EventHubs.Primitives.BlobCheckpointStoreInternal>(),
```

Comment:
This is weird and I didn't notice the namespace change with the rename.    I'll see if I can return to the more readable state.